frontend/javascripts/messages.tsx

```diff
@@ -368,9 +368,11 @@ instead. Only enable this option if you understand its effect. All layers will n
   "dataset.is_scratch":
     "This dataset location is marked as 'scratch' and meant for testing only. Please move this dataset to a permanent storage location and reimport it.",
   "dataset.resolution_mismatch":
-    "This dataset contains multiple layers which differ in their magnification. Please convert the layers to make their resolutions match. Otherwise, rendering errors cannot be avoided.",
+    "This dataset contains multiple layers which differ in their magnification. Please convert the layers to make their magnifications match. Otherwise, rendering errors cannot be avoided.",
   "dataset.z1_downsampling_hint":
     "The currently rendered quality is not optimal due to the available magnifications and the viewport arrangement. To improve the quality try to increase the size of the XY viewport (e.g. by maximizing it).",
+  "dataset.mag_explanation":
+    "Layers contain image data in one or multiple magnifications. The image data in full magnification is referred to as the finest mag, e.g. mag 1-1-1. Magnification 4-4-4 describes a downsampling factor of 4 in each dimension.",
```

Comment:
I am unsure whether "full" in `full magnification` is the correct word here. :thinking: I'd vote for something like this
> The image data in best scanning magnification is referred to as the finest mag, e.g. mag 1-1-1.

I am open to discuss this :)
Because "best scanning magnification" doesn't sound ideal as well :/

Reply:
And maybe extend the explaination a little:

```suggestion
    "Layers contain image data in one or multiple magnifications. The image data in full magnification is referred to as the finest mag, e.g. mag 1-1-1. Magnification 4-4-4 describes a downsampling factor of 4 in each dimension compared to magnification 1-1-1.",
```

Reply:
I think for this one it may be fine to use "resolution", since this is where we explain what we mean with the word magnification/mag. Same as in https://docs.webknossos.org/webknossos/terminology.html

Reply:
"Full resolution" is used in other places too, such as the terminology doc. Maybe "highest resolution" could be an alternative right here? "Best" could mean coarse too, right? Depending on the zoom, for example.

Reply:
(I already committed the changes here that we agree on)

Reply:
@MichaelBuessemeyer what do you think about "highest resolution"? or "full resolution"?

Reply:
I'd say both are fine. You can choose as you want. I might have a very small tendency towards highest, but 
> "Full resolution" is used in other places too, such as the terminology doc.

sounds like a good argument, So let's better keep that :tada: